gradio/components/plot.py

```diff
@@ -39,7 +39,7 @@ class Plot(Component):
     """
 
     data_model = PlotData
-    EVENTS = [Events.change, Events.clear]
+    EVENTS = [Events.change, Events.clear, Events.load]
```

Comment:
Can you explain the rationale for adding the `.load()` event to `gr.Plot()`? Why are we adding `.load()` to a component?

Reply:
Got my wires crossed for a second - backed this out